File: SurgeonPortal.Models/MedicalTraining/OtherCertificationsModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace SurgeonPortal.Models.MedicalTraining
{
    public class OtherCertificationsModel
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int CertificateTypeId { get; set; }
        public string CertificateTypeName { get; set; }
        public DateTime IssueDate { get; set; }
        public string CertificateNumber { get; set; }
        public int CreatedByUserId { get; set; }
        public DateTime CreatedAtUtc { get; set; }
        public DateTime LastUpdatedAtUtc { get; set; }
        public int LastUpdatedByUserId { get; set; }
    }
}

File: SurgeonPortal.Api/ClientApp/src/app/api/models/medicaltraining/other-certifications.model.ts

export interface IOtherCertificationsModel {
    id: number;
    userId: number;
    certificateTypeId: number;
    certificateTypeName: string;
    issueDate: string;
    certificateNumber: string;
    createdByUserId: number;
    createdAtUtc: string;
    lastUpdatedAtUtc: string;
    lastUpdatedByUserId: number;
}
